docs/pages/components/Tooltip.mdx

```diff
@@ -0,0 +1,183 @@
+        Top
+      </Button>
+      <Tooltip target={top} placement="top">
+        On top
+      </Tooltip>
+
+      <Button variant="secondary" buttonRef={bottom}>
+        Bottom
+      </Button>
+      <Tooltip target={bottom} placement="bottom">
+        On bottom
+      </Tooltip>
+
+      <Button variant="secondary" buttonRef={left}>
+        Left
+      </Button>
+      <Tooltip target={left} placement="left">
+        On your left
+      </Tooltip>
+
+      <Button variant="secondary" buttonRef={right}>
+        Right
+      </Button>
+      <Tooltip target={right} placement="right">
+        On your right
+      </Tooltip>
+    </>
+  )
+}
+```
+
+### Info Tooltips
+
+Info Tooltips are an alternate variant to present longer textual content that may occur over multiple lines.
+
+```jsx example
+function InfoTooltipExamples() {
+  const top = useRef();
+  const bottom = useRef();
+  const left = useRef();
+  const right = useRef();
+
+  return (
+    <>
+      <Button variant="secondary" buttonRef={top}>
+        Top
+      </Button>
+      <Tooltip variant="info" target={top} placement="top">
+        Here’s an informational tooltip with more content on top.
+      </Tooltip>
+
+      <Button variant="secondary" buttonRef={bottom}>
+        Bottom
+      </Button>
+      <Tooltip variant="info" target={bottom} placement="bottom">
+        Here’s an informational tooltip with more content on bottom.
+      </Tooltip>
+
+      <Button variant="secondary" buttonRef={left}>
+        Left
+      </Button>
+      <Tooltip variant="info" target={left} placement="left">
+        Here’s an informational tooltip with more content on your left.
+      </Tooltip>
+
+      <Button variant="secondary" buttonRef={right}>
+        Right
+      </Button>
+      <Tooltip variant="info" target={right} placement="right">
+        Here’s an informational tooltip with more content on your right.
+      </Tooltip>
+    </>
+  )
+}
+```
+
+### Big Tooltips
+
+For more verbose tooltips, the `big` variant can be used.
+
+```jsx example
+function BigTooltipExample() {
+  const password = useRef()
+  return (
+    <>
+      <TextField fieldRef={password} label="Password" />
+      <Tooltip variant="big" target={password} placement="auto">
+        <TooltipHead>Password requirements</TooltipHead>
+        <Line />
+        <TooltipContent>
+          <ul>
+            <li>The character length should be long or whatever.</li>
+            <li>
+              You need Lower-case, Upper-case, Super-case, and Basket-case.
+            </li>
+            <li>
+              Special characters are good, but remember all characters are
+              special in their own way
+            </li>
+            <li>
+              Make sure your passwords match...that should be obvious.
+            </li>
+          </ul>
+        </TooltipContent>
+      </Tooltip>
+    </>
+  )
+}
+```
+
+## Props
+
+<ComponentProps
+  children={{
+    required: true
+  }}
+  props={[
```

Comment:
There are a couple of undocumented props: `association` and `hideElementOnHidden`. Is it intentional to leave these undocumented?

Reply:
It wasn't intended as I was porting over the existing documentation but this can be a quick fix here.